packages/app/src/specs/tree/useCollapsibleTree.ts

```diff
@@ -0,0 +1,266 @@
+import type { ComputedRef, Ref } from 'vue'
```

Comment:
Moved from frontend-shared, no changes. This is not a shared file, it is only used in App.